qutip/tests/core/test_states.py

```diff
@@ -33,21 +33,33 @@
 
 import pytest
 import numpy as np
-from numpy.testing import assert_, run_module_suite
+import scipy.sparse as sp
 import qutip
-from qutip import (expect, destroy, coherent, coherent_dm, thermal_dm,
-                   fock_dm, triplet_states)
+from qutip import Qobj
+from qutip.core.states import *
```

Comment:
I don't think we should be using star-imports at all, even in tests - they're very unclear for the reader of the file.